### DISCUSSION TITLE
Access to MARMOT

### DISCUSSION THREAD

**Discussion by PengWei97** [idaholab/moose#16593] [Q&A General] [answered]
Dear MOOSE Developer,
I would like to access MARMOT to study grain growth with Phase field and mechanical coupling model,and be instructed to contact the moose-users group. What should I do, thanks in advance!
Best,
Wei

**Comment by cticenhour** [comment, marked as answer]
Please go to (https://inl.gov/ncrc/) and see the "How NCRC Works" link on that page. Your institution must have a license agreement with INL in place, and all approved users must be evaluated on a case-by-case basis. On that page, there is also an email to submit a question to the NCRC for other questions you may have regarding access.

**Reply by PengWei97** [reply]
Thanks for your good instructions. I will fill in the application materials and look forward to using Marmot.

**Reply by PengWei97** [reply]
Dear cticenhour, 
Thanks for your reply.I read the webpage carefully: INL Modeling and Simulation Software. I still have some doubts and hope to get your answers.@cticenhour 

First, as a PhD student from a Chinese university instead of a native American university, can I get a license to use marmot?  
Second, if I am eligible for a marmot license, what should I or my school pay?  

Kind regards,
Wei Peng
East China University of Science and Technology
School of Mechanical and Power Engineering

**Reply by cticenhour** [reply]
As I am not a member of the access team for MARMOT or our other codes, I am not able to answer these questions. Please submit it to the form on the Access page I sent to you previously, or to the contacts on the "Access Instructions" page linked just above that form.

**Reply by SudiptaBiswas** [reply]
Most of the phase-field models including grain growth are already available in MOOSE. There are examples of phase-field and mechanics coupling in the 'combined' module. You can learn from these examples and build models around them. What do you need from MARMOT specifically?

**Reply by PengWei97** [reply]
My research direction is to study the grain growth and failure mechanism of ultra-fine or nanocrystalline under the interaction of mechanical field and temperature field. I read the article by [L. Zhao](https://www.sciencedirect.com/science/article/abs/pii/S092702561630605X).

> On the plastic driving force of grain boundary migration: A fully coupled phase field and crystal plasticity model

His model is a coupled model of crystal plasticity and phase field, which is simulated by MARMOT. So, using MARMOT may be a good choice for me.
And in the combined example, only the linear elasticity is coupled into the phase field, there is no example of coupled crystal plasticity. I tried to couple j2 into the phase field last month, but failed. #16577
